src/natcap/invest/validation.py

```diff
@@ -6,7 +6,12 @@
 import pprint
 import os
 import re
+import threading
+import functools
 import importlib
+import queue
+import signal
```

Comment:
Is signal unused?  Maybe this was left over from a prototype?

Reply:
oops, forgot to take that out!